modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -5,8 +5,34 @@
 
 This section describes how to manually upgrade your ownCloud installation.
 
-If you’re not comfortable performing a manual upgrade, you can also use your Linux distribution’s
-xref:installation/linux_installation.adoc[package manager].
+[[enable-maintenance-mode]]
+== Enable Maintenance Mode
+
+Put your server in xref:configuration/server/occ_command.adoc#maintenance-commands[maintenance mode] and disable xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[Cron jobs].
+Doing so prevents new logins, locks the sessions of logged-in users, and displays a status screen so that users know what is happening.
+
+There are two ways to enable maintenance mode.
+The preferred method is to use xref:configuration/server/occ_command#maintenance-commands[the occ command] — which you must run as your webserver user.
```

Comment:
I would make the two methods a list item.
`. The preferred method...`
`. The other way...`